src/content/doc-sdk-javascript/data-types.mdx

```diff
@@ -384,10 +384,10 @@ Geometry.coordinates
 
 #### `GeometryPoint`
 
-A point in space, made up of a lat and lon coordinate, automatically converted to a float.
+A point in space, made up of a lon and lat coordinate, automatically converted to a float.
```

Comment:
todo: it is common to name them `lat` and `long`

Reply:
While that's how I refer to them as well, the issue is that we internally save them as lon/lat (in line with the GeoJSON specification), and mentioning these in another order risks confusing users

Reply:
I did not mentioned the order, just the missing `g` in `long` :)

Reply:
Ah yes I see now, good point!